src/lib/auth.ts

```diff
@@ -117,8 +140,8 @@ export const DEVICE_API_KEY_PERMISSIONS = [
 
 ROLES['device-api-key'] = [
 	...DEVICE_API_KEY_PERMISSIONS,
-	`resin.user.get`,
-	`resin.user__has__public_key.get`,
+	`resin.user.read`,
+	`resin.user__has__public_key.read`,
```

Comment:
Not technically necessary, but standardizing on `.read` is nice, and made my grepping far easier ;)